src/style.css

```
@@ -0,0 +1,28 @@
+.starts {
```

I think it is a typo ;)
```suggestion
.stars {
```